storage/sql.go

```diff
@@ -239,6 +240,8 @@ func verifyModel(ctx context.Context, db *pg.DB, m *orm.Table) error {
 			datatype = "timestamptz"
 		} else if datatype == "timestamp without time zone" {
 			datatype = "timestamp"
+		} else if datatype == "USER-DEFINED" && tableName == "miner_sector_events" {
```

Comment:
The query above on line 230 populates `datatype` with the value `USER-DEFINED` which correct but to vague to be of value for validating. I've hard coded this here since its the only user defined type we have, open to alternatives.

Reply:
I think think the `udt_name` column contains the name of the user defined type (udt=user defined type)

Reply:
That did the trick, thanks.